src/interactiveConcatTextDocument.ts

```diff
@@ -3,333 +3,288 @@
 /* eslint-disable class-methods-use-this */
 
 import {
-  NotebookDocument,
-  Position,
-  Range,
-  TextDocument,
-  Uri,
-  workspace,
-  Event,
-  EventEmitter,
-  Location,
-  TextLine,
-} from "vscode";
-import { NotebookConcatTextDocument } from "vscode";
-
-import { IVSCodeNotebook } from "./common/types";
-import { InteractiveInputScheme } from "./common/utils";
-import { IConcatTextDocument, score } from "./concatTextDocument";
+    NotebookDocument,

```

Comment:
Sorry this file diff is a bit hard to read. Are there actual changes here? Looks like mostly moves and formatting.

Reply:
That file was just a reformat. I added the selector change to that file but Peng's formatter is different than what we have for jupyter, so prettier rewrote most of the file.